app/styles/_core.less

```diff
@@ -1161,10 +1161,19 @@ copy-to-clipboard .input-group.limit-width {
   justify-content: space-between;
 }
 
-
 // add an option for a single-column display at screen resolutions 0-479px
 @media (max-width: @screen-xxs-max) {
   .col-xxs-12 {
     width: 100%;
   }
 }
+
+// fixes wobble that occurs when status-icons utilize fa-spin
+// since font awesome icons are created on a 14px grid, setting the font-size to 14px improves anti-aliasing
+// line-height: normal fixes the wobble (see https://github.com/FortAwesome/Font-Awesome/issues/671#issuecomment-249591495 )
+// margin-right puts a bit of space between the icon and the text that follows, which is necessary given the larger icon size
+.status-icon > .fa-spin {
+  font-size: 14px;
```

Comment:
any reason not to make all the status icons 14px? not just when fa-spin is there